docs/guides/dynamo_deploy/dynamo_cloud.md

```diff
@@ -190,3 +190,40 @@ After deploying the Dynamo cloud platform, you can:
 3. Manage your deployments using the Dynamo CLI
 
 For more detailed information about deploying inference graphs, see the [Dynamo Deploy Guide](README.md).
+
+
+### Installation using published helm chart
+
+To install Dynamo Cloud using the published Helm chart, you'll need to set the required environment variables and use Helm's `--set` flags to pass your configuration values.
+First, export your environment variables (as described in previous paragraph):
+
+```bash
+export DOCKER_SERVER=<your-docker-server>
+export DOCKER_SECRET_NAME=<your-docker-secret-name> # Allowing to pull images from DOCKER_SERVER
+export IMAGE_TAG=<TAG>  # Use the same tag you used when building the images
+export PIPELINES_DOCKER_SERVER=<your-docker-server>
+export PIPELINES_DOCKER_USERNAME=<your-docker-username>
+export PIPELINES_DOCKER_PASSWORD=<your-docker-password>
```

Comment:
this is scary. why can't this rely on the Kube secret alone?

additionally, it puts a secret into the environment which is unsafe.